lib/Basic/LangOptions.cpp

```diff
@@ -101,6 +101,8 @@ std::pair<bool, bool> LangOptions::setTarget(llvm::Triple triple) {
     addTargetConfigOption("os", "Linux");
   else if (triple.isOSFreeBSD())
     addTargetConfigOption("os", "FreeBSD");
+  else if (triple.isOSWindows())
+    addTargetConfigOption("os", "Windows");
```

Comment:
For this, please add tests like `test/BuildConfigurations/x64LinuxTarget.swift` for every new supported triple.